lib/util/template/index.ts

```diff
@@ -13,7 +13,7 @@ handlebars.registerHelper('stringToPrettyJSON', (input: string): string =>
 // istanbul ignore next
 handlebars.registerHelper(
   'replace',
-  (find, replace, context) => context.replace(new RegExp(find, 'g'), replace) // TODO #12070
+  (find, replace, context) => context.replace(find, 'g', replace) // TODO #12070 adding regex here causes coverage issues
```

Comment:
i think this change is wrong and will break existing users, so revert
```suggestion
  (find, replace, context) => context.replace(new RegExp(find, 'g'), replace) // TODO #12070 adding regex here causes coverage issues
```